javascript/index.js

```diff
@@ -14,52 +14,90 @@ const milUniElement = document.getElementById('milUni');
 const splitsElement = document.getElementById('splits');
 
```

Comment:
@TA-Remote is it good practice when working with DOM elements to define them as global variables? or is it just for simplification in the lab and would otherwise be called global object pollution 😜

Reply:
If you are reusing that variable, in this case you need to have access to it in several functions - it's not pollution! And it also keeps code organised. When you start reading the file from top->down, you can see all references in one place